addon/components/frost-table-cell-renderer.js

```diff
@@ -14,8 +14,12 @@ export default Component.extend({
   // == PropTypes =============================================================
 
   propTypes: {
-    // options
+    // optional
+    colIndex: PropTypes.number,
```

Comment:
It's not -- good catch. I had added that at one point, and then later realized that ember lets me avoid needing to tell cells their coordinates.  If I give it the column index, I may as well give it the whole column object, but I can't imagine why we'd need the cell renderer to know that -- it just needs data and a way to propagage actions, I think.

I'll remove that :)